src/test/kotlin/io/github/mojira/arisa/modules/CrashModuleTest.kt

```diff
@@ -707,20 +789,54 @@ class CrashModuleTest : StringSpec({
             priority = NoPriority,
             resolveAsDuplicate = { resolvedAsDupe = true },
             resolveAsInvalid = { resolvedAsInvalid = true },
-            addComment = { addedComment = it }
+            addComment = { addedComment = it },
+            addAttachment = { addedAttachement = true }
         )
 
         val result = module(issue, A_SECOND_AGO)
 
         result.shouldBeRight(ModuleResponse)
         resolvedAsDupe.shouldBeTrue()
+        addedAttachement.shouldBeFalse()
         resolvedAsInvalid.shouldBeFalse()
         addedComment shouldBe CommentOptions("duplicate-tech", "MC-32606")
     }
 
+    "should add attachment when deobfuscated" {
+        var resolvedAsDupe = false
+        var resolvedAsInvalid = false
+        var addedAttachement = false
+
+        val module = CrashModule(
+            listOf("txt"),
+            listOf(CrashDupeConfig("java", "ig75icd64\\.dll", "MC-32606")),
+            crashReader,
+            "duplicate-tech",
+            "modified-game"
+        )
+        val issue = mockIssue(
+            attachments = listOf(getAttachment(OBFUSCATED_CRASH)),
+            description = OBFUSCATED_CRASH,
+            created = NOW,
+            confirmationStatus = Unconfirmed,
+            priority = NoPriority,
+            resolveAsDuplicate = { resolvedAsDupe = true },
+            resolveAsInvalid = { resolvedAsInvalid = true },
+            addAttachment = { addedAttachement = true }
+        )
+
+        val result = module(issue, A_SECOND_AGO)
+
+        result.shouldBeLeft(OperationNotNeededModuleResponse)
```

Comment:
The `OperationNotNeededModuleResponse` irks me a bit here... but out of scope for this PR